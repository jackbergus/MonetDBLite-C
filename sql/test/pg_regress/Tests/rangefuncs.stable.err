stderr of test 'rangefuncs` in directory 'sql/test/pg_regress` itself:


# 17:11:07 >  
# 17:11:07 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=38959" "--set" "mapi_usock=/var/tmp/mtest-1142/.s.monetdb.38959" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/ufs/dinther/INSTALL/var/MonetDB/mTests_sql_test_pg_regress" "--set" "mal_listing=0"
# 17:11:07 >  

# builtin opt 	gdk_dbpath = /ufs/dinther/INSTALL/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 38959
# cmdline opt 	mapi_usock = /var/tmp/mtest-1142/.s.monetdb.38959
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	gdk_dbpath = /ufs/dinther/INSTALL/var/MonetDB/mTests_sql_test_pg_regress
# cmdline opt 	mal_listing = 0
# cmdline opt 	gdk_debug = 536870922

# 17:11:09 >  
# 17:11:09 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-1142" "--port=38959"
# 17:11:09 >  

MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT string, setting FROM pg_settings WHERE name LIKE 'enable%';
ERROR = !syntax error, unexpected CLOB in: "select string"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE FUNCTION foot(int) returns setof foo2 as 'SELECT * FROM foo2 WHERE fooid = $1;' LANGUAGE SQL;
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' in: "create function foot(int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = select * from foo2, foot(foo2.fooid) z where foo2.f2 = z.f2;
ERROR = !FOOT: missing group by
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = select * from foo2 where f2 in (select f2 from foot(foo2.fooid) z where z.fooid = foo2.fooid) ORDER BY 1,2;
ERROR = !SELECT: no such unary operator 'foot(int)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = select * from foo2 where f2 in (select f2 from foot(1) z where z.fooid = foo2.fooid) ORDER BY 1,2;
ERROR = !SELECT: no such unary operator 'foot(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = select * from foo2 where f2 in (select f2 from foot(foo2.fooid) z where z.fooid = 1) ORDER BY 1,2;
ERROR = !SELECT: no such unary operator 'foot(int)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = select foot.fooid, foot.f2 from foot(sin(pi()/2)::int) ORDER BY 1,2;
ERROR = !unexpected symbol (:)
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE FUNCTION getfoo(int) RETURNS int AS 'SELECT $1;' LANGUAGE SQL;
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' in: "create function getfoo(int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM getfoo(1) AS t1;
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE VIEW vw_getfoo AS SELECT * FROM getfoo(1);
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM vw_getfoo;
ERROR = !SELECT: no such table 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP VIEW vw_getfoo;
ERROR = !DROP VIEW: unknown view 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP FUNCTION getfoo(int);
ERROR = !DROP FUNCTION: no such function 'getfoo' (int(32))
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE FUNCTION getfoo(int) RETURNS setof int AS 'SELECT fooid FROM foo WHERE fooid = $1;' LANGUAGE SQL;
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' in: "create function getfoo(int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM getfoo(1) AS t1;
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE VIEW vw_getfoo AS SELECT * FROM getfoo(1);
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM vw_getfoo;
ERROR = !SELECT: no such table 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP VIEW vw_getfoo;
ERROR = !DROP VIEW: unknown view 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP FUNCTION getfoo(int);
ERROR = !DROP FUNCTION: no such function 'getfoo' (int(32))
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE FUNCTION getfoo(int) RETURNS setof text AS 'SELECT fooname FROM foo WHERE fooid = $1;' LANGUAGE SQL;
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' in: "create function getfoo(int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM getfoo(1) AS t1;
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE VIEW vw_getfoo AS SELECT * FROM getfoo(1);
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM vw_getfoo;
ERROR = !SELECT: no such table 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP VIEW vw_getfoo;
ERROR = !DROP VIEW: unknown view 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP FUNCTION getfoo(int);
ERROR = !DROP FUNCTION: no such function 'getfoo' (int(32))
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE FUNCTION getfoo(int) RETURNS foo AS 'SELECT * FROM foo WHERE fooid = $1;' LANGUAGE SQL;
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' in: "create function getfoo(int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM getfoo(1) AS t1;
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE VIEW vw_getfoo AS SELECT * FROM getfoo(1);
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM vw_getfoo;
ERROR = !SELECT: no such table 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP VIEW vw_getfoo;
ERROR = !DROP VIEW: unknown view 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP FUNCTION getfoo(int);
ERROR = !DROP FUNCTION: no such function 'getfoo' (int(32))
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE FUNCTION getfoo(int) RETURNS setof foo AS 'SELECT * FROM foo WHERE fooid = $1;' LANGUAGE SQL;
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' in: "create function getfoo(int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM getfoo(1) AS t1;
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE VIEW vw_getfoo AS SELECT * FROM getfoo(1);
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM vw_getfoo;
ERROR = !SELECT: no such table 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP VIEW vw_getfoo;
ERROR = !DROP VIEW: unknown view 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP FUNCTION getfoo(int);
ERROR = !DROP FUNCTION: no such function 'getfoo' (int(32))
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE FUNCTION getfoo(int) RETURNS RECORD AS 'SELECT * FROM foo WHERE fooid = $1;' LANGUAGE SQL;
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' in: "create function getfoo(int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM getfoo(1) AS t1(fooid int, foosubid int, fooname text);
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' or ',' in: "select * from getfoo(1) as t1(fooid int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE VIEW vw_getfoo AS SELECT * FROM getfoo(1) AS 
        (fooid int, foosubid int, fooname text);
ERROR = !syntax error, unexpected '(' in: "create view vw_getfoo as select * from getfoo(1) as 
        !("
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM vw_getfoo;
ERROR = !SELECT: no such table 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP VIEW vw_getfoo;
ERROR = !DROP VIEW: unknown view 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP FUNCTION getfoo(int);
ERROR = !DROP FUNCTION: no such function 'getfoo' (int(32))
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE FUNCTION getfoo(int) RETURNS setof record AS 'SELECT * FROM foo WHERE fooid = $1;' LANGUAGE SQL;
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' in: "create function getfoo(int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM getfoo(1) AS t1(fooid int, foosubid int, fooname text);
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' or ',' in: "select * from getfoo(1) as t1(fooid int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE VIEW vw_getfoo AS SELECT * FROM getfoo(1) AS
        (fooid int, foosubid int, fooname text);
ERROR = !syntax error, unexpected '(' in: "create view vw_getfoo as select * from getfoo(1) as
        !("
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM vw_getfoo;
ERROR = !SELECT: no such table 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP VIEW vw_getfoo;
ERROR = !DROP VIEW: unknown view 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP FUNCTION getfoo(int);
ERROR = !DROP FUNCTION: no such function 'getfoo' (int(32))
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE FUNCTION getfoo(int) RETURNS int AS 'DECLARE fooint int; BEGIN SELECT fooid into fooint FROM foo WHERE fooid = $1; RETURN fooint; COMMIT;' LANGUAGE 'plpgsql';
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' in: "create function getfoo(int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM getfoo(1) AS t1;
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE VIEW vw_getfoo AS SELECT * FROM getfoo(1);
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM vw_getfoo;
ERROR = !SELECT: no such table 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP VIEW vw_getfoo;
ERROR = !DROP VIEW: unknown view 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP FUNCTION getfoo(int);
ERROR = !DROP FUNCTION: no such function 'getfoo' (int(32))
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE FUNCTION getfoo(int) RETURNS foo AS 'DECLARE footup foo%ROWTYPE; BEGIN SELECT * into footup FROM foo WHERE fooid = $1; RETURN footup; COMMIT;' LANGUAGE 'plpgsql';
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' in: "create function getfoo(int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM getfoo(1) AS t1;
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE VIEW vw_getfoo AS SELECT * FROM getfoo(1);
ERROR = !SELECT: no such unary operator 'getfoo(tinyint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM vw_getfoo;
ERROR = !SELECT: no such table 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP VIEW vw_getfoo;
ERROR = !DROP VIEW: unknown view 'vw_getfoo'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP FUNCTION getfoo(int);
ERROR = !DROP FUNCTION: no such function 'getfoo' (int(32))
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP FUNCTION foot(int);
ERROR = !DROP FUNCTION: no such function 'foot' (int(32))
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE FUNCTION foorescan(int,int) RETURNS setof foorescan AS 'SELECT * FROM foorescan WHERE fooid >= $1 and fooid < $2 ;' LANGUAGE SQL;
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' in: "create function foorescan(int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM foorescan f WHERE f.fooid IN (SELECT fooid FROM foorescan(5002,5004)) ORDER BY 1,2;
ERROR = !SELECT: no such binary operator 'foorescan(smallint,smallint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE VIEW vw_foorescan AS SELECT * FROM foorescan(5002,5004);
ERROR = !SELECT: no such binary operator 'foorescan(smallint,smallint)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM foorescan f WHERE f.fooid IN (SELECT fooid FROM vw_foorescan) ORDER BY 1,2;
ERROR = !SELECT: no such table 'vw_foorescan'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE FUNCTION foorescan(int) RETURNS setof foorescan AS 'SELECT * FROM foorescan WHERE fooid = $1;' LANGUAGE SQL;
ERROR = !syntax error, unexpected sqlINTEGER, expecting ')' in: "create function foorescan(int"
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT f.* FROM barrescan b, foorescan f WHERE f.fooid = b.fooid AND b.fooid IN (SELECT fooid FROM foorescan(b.fooid)) ORDER BY 1,2;
ERROR = !SELECT: no such unary operator 'foorescan(int)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT b.fooid, max(f.foosubid) FROM barrescan b, foorescan f WHERE f.fooid = b.fooid AND b.fooid IN (SELECT fooid FROM foorescan(b.fooid)) GROUP BY b.fooid ORDER BY 1,2;
ERROR = !SELECT: no such unary operator 'foorescan(int)'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE VIEW fooview1 AS SELECT f.* FROM barrescan b, foorescan f WHERE f.fooid = b.fooid AND b.fooid IN (SELECT fooid FROM foorescan(b.fooid)) ORDER BY 1,2;
ERROR = !CREATE VIEW: ORDER BY not supported
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM fooview1 AS fv WHERE fv.fooid = 5004;
ERROR = !SELECT: no such table 'fooview1'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = CREATE VIEW fooview2 AS SELECT b.fooid, max(f.foosubid) AS maxsubid FROM barrescan b, foorescan f WHERE f.fooid = b.fooid AND b.fooid IN (SELECT fooid FROM foorescan(b.fooid)) GROUP BY b.fooid ORDER BY 1,2;
ERROR = !CREATE VIEW: ORDER BY not supported
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = SELECT * FROM fooview2 AS fv WHERE fv.maxsubid = 5;
ERROR = !SELECT: no such table 'fooview2'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP VIEW vw_foorescan;
ERROR = !DROP VIEW: unknown view 'vw_foorescan'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP VIEW fooview1;
ERROR = !DROP VIEW: unknown view 'fooview1'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP VIEW fooview2;
ERROR = !DROP VIEW: unknown view 'fooview2'
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP FUNCTION foorescan(int,int);
ERROR = !DROP FUNCTION: no such function 'foorescan' (int(32), int(32))
MAPI  = (monetdb) /var/tmp/mtest-1142/.s.monetdb.38959
QUERY = DROP FUNCTION foorescan(int);
ERROR = !DROP FUNCTION: no such function 'foorescan' (int(32))

# 17:11:10 >  
# 17:11:10 >  "Done."
# 17:11:10 >  

