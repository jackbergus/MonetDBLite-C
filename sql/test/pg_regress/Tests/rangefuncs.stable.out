stdout of test 'rangefuncs` in directory 'sql/test/pg_regress` itself:


# 17:11:07 >  
# 17:11:07 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=38959" "--set" "mapi_usock=/var/tmp/mtest-1142/.s.monetdb.38959" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/ufs/dinther/INSTALL/var/MonetDB/mTests_sql_test_pg_regress" "--set" "mal_listing=0"
# 17:11:07 >  

# MonetDB 5 server v11.18.0
# This is an unreleased version
# Serving database 'mTests_sql_test_pg_regress', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 15.356 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2014 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://uwakai.da.cwi.nl:38959/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-1142/.s.monetdb.38959
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

Ready.

# 17:11:09 >  
# 17:11:09 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-1142" "--port=38959"
# 17:11:09 >  

#CREATE TABLE foo2(fooid int, f2 int);
#INSERT INTO foo2 VALUES(1, 11);
[ 1	]
#INSERT INTO foo2 VALUES(2, 22);
[ 1	]
#INSERT INTO foo2 VALUES(1, 111);
[ 1	]
#CREATE TABLE foo (fooid int, foosubid int, fooname text, primary key(fooid,foosubid));
#INSERT INTO foo VALUES(1,1,'Joe');
[ 1	]
#INSERT INTO foo VALUES(1,2,'Ed');
[ 1	]
#INSERT INTO foo VALUES(2,1,'Mary');
[ 1	]
#DROP TABLE foo2;
#DROP TABLE foo;
#CREATE TABLE foorescan (fooid int, foosubid int, fooname text, primary key(fooid,foosubid));
#INSERT INTO foorescan values(5000,1,'abc.5000.1');
[ 1	]
#INSERT INTO foorescan values(5001,1,'abc.5001.1');
[ 1	]
#INSERT INTO foorescan values(5002,1,'abc.5002.1');
[ 1	]
#INSERT INTO foorescan values(5003,1,'abc.5003.1');
[ 1	]
#INSERT INTO foorescan values(5004,1,'abc.5004.1');
[ 1	]
#INSERT INTO foorescan values(5005,1,'abc.5005.1');
[ 1	]
#INSERT INTO foorescan values(5006,1,'abc.5006.1');
[ 1	]
#INSERT INTO foorescan values(5007,1,'abc.5007.1');
[ 1	]
#INSERT INTO foorescan values(5008,1,'abc.5008.1');
[ 1	]
#INSERT INTO foorescan values(5009,1,'abc.5009.1');
[ 1	]
#INSERT INTO foorescan values(5000,2,'abc.5000.2');
[ 1	]
#INSERT INTO foorescan values(5001,2,'abc.5001.2');
[ 1	]
#INSERT INTO foorescan values(5002,2,'abc.5002.2');
[ 1	]
#INSERT INTO foorescan values(5003,2,'abc.5003.2');
[ 1	]
#INSERT INTO foorescan values(5004,2,'abc.5004.2');
[ 1	]
#INSERT INTO foorescan values(5005,2,'abc.5005.2');
[ 1	]
#INSERT INTO foorescan values(5006,2,'abc.5006.2');
[ 1	]
#INSERT INTO foorescan values(5007,2,'abc.5007.2');
[ 1	]
#INSERT INTO foorescan values(5008,2,'abc.5008.2');
[ 1	]
#INSERT INTO foorescan values(5009,2,'abc.5009.2');
[ 1	]
#INSERT INTO foorescan values(5000,3,'abc.5000.3');
[ 1	]
#INSERT INTO foorescan values(5001,3,'abc.5001.3');
[ 1	]
#INSERT INTO foorescan values(5002,3,'abc.5002.3');
[ 1	]
#INSERT INTO foorescan values(5003,3,'abc.5003.3');
[ 1	]
#INSERT INTO foorescan values(5004,3,'abc.5004.3');
[ 1	]
#INSERT INTO foorescan values(5005,3,'abc.5005.3');
[ 1	]
#INSERT INTO foorescan values(5006,3,'abc.5006.3');
[ 1	]
#INSERT INTO foorescan values(5007,3,'abc.5007.3');
[ 1	]
#INSERT INTO foorescan values(5008,3,'abc.5008.3');
[ 1	]
#INSERT INTO foorescan values(5009,3,'abc.5009.3');
[ 1	]
#INSERT INTO foorescan values(5000,4,'abc.5000.4');
[ 1	]
#INSERT INTO foorescan values(5001,4,'abc.5001.4');
[ 1	]
#INSERT INTO foorescan values(5002,4,'abc.5002.4');
[ 1	]
#INSERT INTO foorescan values(5003,4,'abc.5003.4');
[ 1	]
#INSERT INTO foorescan values(5004,4,'abc.5004.4');
[ 1	]
#INSERT INTO foorescan values(5005,4,'abc.5005.4');
[ 1	]
#INSERT INTO foorescan values(5006,4,'abc.5006.4');
[ 1	]
#INSERT INTO foorescan values(5007,4,'abc.5007.4');
[ 1	]
#INSERT INTO foorescan values(5008,4,'abc.5008.4');
[ 1	]
#INSERT INTO foorescan values(5009,4,'abc.5009.4');
[ 1	]
#INSERT INTO foorescan values(5000,5,'abc.5000.5');
[ 1	]
#INSERT INTO foorescan values(5001,5,'abc.5001.5');
[ 1	]
#INSERT INTO foorescan values(5002,5,'abc.5002.5');
[ 1	]
#INSERT INTO foorescan values(5003,5,'abc.5003.5');
[ 1	]
#INSERT INTO foorescan values(5004,5,'abc.5004.5');
[ 1	]
#INSERT INTO foorescan values(5005,5,'abc.5005.5');
[ 1	]
#INSERT INTO foorescan values(5006,5,'abc.5006.5');
[ 1	]
#INSERT INTO foorescan values(5007,5,'abc.5007.5');
[ 1	]
#INSERT INTO foorescan values(5008,5,'abc.5008.5');
[ 1	]
#INSERT INTO foorescan values(5009,5,'abc.5009.5');
[ 1	]
#CREATE TABLE barrescan (fooid int primary key);
#INSERT INTO barrescan values(5003);
[ 1	]
#INSERT INTO barrescan values(5004);
[ 1	]
#INSERT INTO barrescan values(5005);
[ 1	]
#INSERT INTO barrescan values(5006);
[ 1	]
#INSERT INTO barrescan values(5007);
[ 1	]
#INSERT INTO barrescan values(5008);
[ 1	]
#DROP TABLE foorescan;
#DROP TABLE barrescan;

# 17:11:10 >  
# 17:11:10 >  "Done."
# 17:11:10 >  

